packages/loot-core/package.json

```diff
@@ -41,6 +41,7 @@
   "devDependencies": {
     "@actual-app/api": "workspace:^",
     "@actual-app/crdt": "workspace:^",
+    "@actual-app/web": "workspace:^",
```

Comment:
This is not great 😬 
(but I get that your changes in this PR don't actually introduce this)